man/man8/cpudist.8

```diff
@@ -70,7 +70,7 @@ Print 1 second summaries, using milliseconds as units for the histogram, and inc
 .TP
 Trace PID 186 only, 1 second summaries:
 #
-.B cpudist -P 185 1
+.B cpudist -p 185 1
```

Comment:
`185` here does not match the above description, please fix it.

Reply:
fixed